src/main/java/org/spongepowered/common/mixin/command/MixinCommandToggleDownfall.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.mixin.command;
+
+import net.minecraft.command.CommandToggleDownfall;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.world.storage.WorldInfo;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+@Mixin(CommandToggleDownfall.class)
+public abstract class MixinCommandToggleDownfall {
+
+    @Redirect(method = "processCommand", at = @At(value = "INVOKE",
+            target = "Lnet/minecraft/command/CommandToggleDownfall;toggleDownfall()V"))
+    private void onToggleDownfall(CommandToggleDownfall this$0, ICommandSender sender, String[] args) {
```

Comment:
It's an unneeded parameter, because the context is already available as the current instance, so it could be any name. If I remember correctly we had a weird name that was normally used for these kind of parameters, couldn't remember it correctly anymore though :P

Just tell me how to name it and I'll change it, I don't care because it is never used anyway.